concrete/views/dialogs/user/details.php

```diff
@@ -0,0 +1,82 @@
+<?php
+defined('C5_EXECUTE') or die("Access Denied.");
+?>
+
+<div class="ccm-ui">
+
+
+    <div class="form-group">
+        <div>
+            <label class="control-label"><?= t('Username') ?></label>
+        </div>
+        <div>
+            <?= $uName ?>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <div>
+            <label class="control-label"><?= t('Email') ?></label>
+        </div>
+        <div>
+            <?= $uEmail ?>
+        </div>
+    </div>
+
+    <!-- user group starts -->
+    <?php if (count($userGroups) > 0) { ?>
+        <h4><?= t('Groups') ?></h4>
+        <ul class="list-unstyled">
+            <?php foreach ($userGroups as $group) { ?>
+                <li><?= $group->getGroupDisplayName() ?></li>
+            <?php } ?>
+        </ul>
+    <?php } ?>
+    <!-- user group ends -->
+
+    <!-- user attribut starts -->
+    <?php if (count($attributeSets) > 0) : ?>
+        <h4><?php echo t('User Attributes') ?></h4>
+        <br>
+        <?php foreach ($attributeSets as $set) { ?>
+            <h5><?php echo $set->getAttributeSetDisplayName() ?></h5>
+            <?php foreach ($set->getAttributeKeys() as $ak) {
+                $value = $user->getAttributeValueObject($ak);
+                ?>
+                <div class="form-group">
+                    <div>
+                        <label class="control-label"><?= $ak->getAttributeKeyDisplayName() ?></label>
+                    </div>
+                    <div>
+                        <?= $value ?>
```

Comment:
@aembler I think many of the values in this file need to be escaped in order to prevent XSS.

Reply:
Casting attribute values to string should run them through getDisplayValue(), which should be sanitized.

Reply:
@aembler Yes, you are right. 👍